tests/MongoDB.Driver.Core.Tests/Core/Operations/CreateIndexesUsingCommandOperationTests.cs

```diff
@@ -187,7 +187,6 @@ public void CreateCommand_should_return_expected_result_when_MaxTime_is_Set(long
             [Values(false, true)]
             bool async)
         {
-            RequireServer.Check().Supports(Feature.CreateIndexesCommand);
```

Comment:
Keep the RequireServer.Check() part

Same for all other methods were this happened.

Reply:
Done for all removed `RequireServer.Check();` occurrences.